Resources/translations/SonataUserBundle.de.xliff

```diff
@@ -514,10 +514,6 @@
                 <source>Gender</source>
                 <target>Geschlecht</target>
             </trans-unit>
-            <trans-unit id="Bad credentials">
-                <source>Bad credentials</source>
-                <target>Benutzername oder Passwort ungültig</target>
-            </trans-unit>
```

Comment:
This translation was only a quickfix to the problem, and it would not work perfectly.
Now the exception message is translated with the security catalogue, so is Symfony who is handling the translation.